README.md

```diff
@@ -32,6 +34,8 @@ For commodity purpose, the Collection comes with a role 'jcliff' to have Ansible
       tasks:
         ...
 
+Refers to [JCliff](https://github.com/bserdar/jcliff) for more information on how to install the tool manually. JCliff being a Java based application setting up is pretty simple, so do not let it deter you from using the collection.
```

Comment:
The github project you refers here don t at all detail how to build jcliff and launch it. There is, at most, a bash script which can be use to launch it if it has been compiled as tar - https://github.com/bserdar/jcliff/blob/master/jcliff-dev.sh#L24

Reply:
That's a good point. It's a very simple Java CLI tool, so I think @bserdar never felt the need to write up the installation instruction. I've did then the RPM, but you are right a few simple steps to do the install completely manually would be nice to have. I'll set up a PR on bserdar's repo to do just that.